src/libnnstreamer-edge/nnstreamer-edge-mqtt-paho.c

```diff
@@ -258,6 +258,8 @@ nns_edge_mqtt_close (nns_edge_broker_h broker_h)
     MQTTAsync_destroy (&handle);
   }
 
+  bh->mqtt_h = NULL;
```

Comment:
What about ```bh->connected``` ?

Reply:
The struct for paho-mqtt does not contain connected flag.